src/meta/processors/job/MetaJobExecutor.cpp

```diff
@@ -211,15 +211,15 @@ nebula::cpp2::ErrorCode MetaJobExecutor::execute() {
     }
   }
 
-  std::vector<folly::SemiFuture<Status>> futs;
+  std::vector<folly::SemiFuture<Status>> futureStatus;
```

Comment:
I'm not sure about this, but there's really no particular reason to abbreviate (see the preceding hunk which has `hostAndPart` and not `hap` or something...

Reply:
IMO, "futs" is just means "futures" rather than "future status".